### PR TITLE
Hide getErrorAttachment callback since backend doesn't support the feature

### PR DESCRIPTION
**Pull request by jaelim-ms**

### CODE REVIEW

File: sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/ErrorAttachments.java [hunk 1]
```diff
@@ -13,6 +12,7 @@
  * Error attachment utilities.
  */
 @SuppressWarnings("WeakerAccess")
+/* TODO (getErrorAttachment): Re-enable error attachment in javadoc when the feature becomes available. Add @ before link. */
 public final class ErrorAttachments {
```
Comment (guperrot) [inline sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/ErrorAttachments.java]: Javadoc tool will still produce documentation for any public class, I really don't understand the changes on this class. We should rather change visibility of this class to package private and make necessary adjustement (visibility tricks) if we use it for testing somewhere.

Comment (jaelim-ms) [inline sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/ErrorAttachments.java]: Oops, missed that javadoc thing. Changing it to package level. This is enough not to have `ErrorAttachments` javadoc.

Comment (guperrot) [inline sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/ErrorAttachments.java]: You didn't restore the @ character in links.

File: sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/ErrorAttachments.java [hunk 2]
```diff
@@ -13,7 +12,8 @@
  * Error attachment utilities.
  */
 @SuppressWarnings("WeakerAccess")
-public final class ErrorAttachments {
+/* TODO (getErrorAttachment): Re-enable error attachment in javadoc when the feature becomes available. Add @ before link. */
```
Comment (guperrot) [inline sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/ErrorAttachments.java]: Remove the comment about `Add @ before link`

File: sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/ErrorAttachments.java [hunk 3]
```diff
@@ -22,7 +22,7 @@
     }
 
     /**
-     * Build an attachment with text suitable for using in {@link CrashesListener#getErrorAttachment(ErrorReport)}.
+     * Build an attachment with text suitable for using in {link CrashesListener#getErrorAttachment(ErrorReport)}.
```
Comment (guperrot) [inline sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/ErrorAttachments.java]: Restore all @ characters

Comment (jaelim-ms) [inline sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/ErrorAttachments.java]: Nope. That is a compile error.

File: sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/ErrorAttachments.java (hunk 2 above)
Comment (jaelim-ms) [inline sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/ErrorAttachments.java]: Look at an answer of your another feedback.

**Pull request merged**